### PR TITLE
Uprev to align with tpm2-tss 2.0

### LINKED ISSUE #2: Faild to build with latest meta-secure-core
The meta-secure-core commit dc173c7a8dff4ac28ea0b075baa664aee5b92e6b updated tpm2 related packages from 1.x to 2.x which makes the cryptfs-tpm2 build failed:

In file included from init.c:34:
/buildarea/build/tmp/work/i586-poky-linux/cryptfs-tpm2/0.6.3+gitAUTOINC+fb49abad57-r0/git/src/include/cryptfs_tpm2.h:64:10: fatal error: tcti/tcti_socket.h: No such file or directory
 #include <tcti/tcti_socket.h>
          ^~~~~~~~~~~~~~~~~~~~
compilation terminated.
<builtin>: recipe for target 'init.o' failed
make[2]: *** [init.o] Error 1

### PR DESCRIPTION
**Pull request by yunguowei**
This patchset fixes #2 

Will submit cryptfs-tpm bb file changes in meta-secure-layer later.

Yunguo

### CODE REVIEW

**Comment by yunguowei** [comment]
@jiazhang0 Any comment on this thread?

File: scripts/luks-setup.sh [hunk 1]
```diff
@@ -100,6 +100,9 @@ detect_tpm() {
 
         grep -q "TPM 2.0 Device" "/sys/class/tpm/$dev/device/description" 2>/dev/null &&
             tpm_absent=0 && break
+
+	grep -q "TPM 2.0 Device" "/sys/class/tpm/$dev/device/firmware_node/description" 2>/dev/null &&
```
Comment (jiazhang0) [inline scripts/luks-setup.sh:104]: Fix this indent.

File: NEWS [hunk 2]
```diff
@@ -1,4 +1,10 @@
 Version 0.6.2
+Version 0.7.0
```
Comment (jiazhang0) [inline NEWS:2]: This line should be on top of 0.6.2.

**Comment by jiazhang0** [comment]
Basically this patch looks good. But we need to keep the compatibility with 1.x. Please enclose the codes for 1.x with a macro like TSS2_LEGACY_V1 or similar and allow to change it in Makefile.

**Comment by yunguowei** [comment]
Thanks for the review, I will send v2 right soob.

**Pull request closed without merging**